thirdparty/freetype2/CMakeLists.txt

```diff
@@ -19,11 +19,7 @@ set(CFG_OPTS "-q --prefix=${BINARY_DIR} --disable-static --enable-shared --with-
 set(CFG_CMD sh -c "${CFG_ENV_VAR} ${SOURCE_DIR}/configure ${CFG_OPTS}")
 
 if($ENV{ANDROID})
-    set(CFG_CMD "${CFG_CMD} && ${ISED} 's|version_type=none|version_type=linux|' libtool")
-    set(CFG_CMD "${CFG_CMD} && ${ISED} 's|need_lib_prefix=no|need_lib_prefix=yes|' libtool")
-    set(CFG_CMD "${CFG_CMD} && ${ISED} 's|need_version=no|need_version=yes|' libtool")
-    set(CFG_CMD "${CFG_CMD} && ${ISED} 's|library_names_spec=.*|library_names_spec=\"\\\\\$libname\\\\\$release\\\\\$shared_ext\\\\\$versuffix \\\\\$libname\\\\\$release\\\\\$shared_ext\\\\\$major \\\\\$libname\\\\\$shared_ext\"|' libtool")
-    set(CFG_CMD "${CFG_CMD} && ${ISED} 's|soname_spec=.*|soname_spec=\"\\\\\$libname\\\\\$release\\\\\$shared_ext\\\\\$major\"|' libtool")
+    set(CFG_CMD "${CFG_CMD} && ${ISED} 's|\${shared_ext}\\\\\$major|\${shared_ext}|' libtool")
```

Comment:
They're not using standard libtool?

Reply:
No idea. I reverted https://github.com/koreader/koreader-base/pull/521/files#diff-73c14d25846120511b077be619c1196ebe5e91a33f21fd4c349e6b9bcf2d6d4a

:see_no_evil:

Reply:
In that case those lines are probably not doing anything, meant for the version in Ubuntu 14.04.

Reply:
Yup, " [fix] patch libtool version < 2.4.6 to generate lib without version suffix for Android"
https://github.com/koreader/koreader-base/commit/48a2ff599799cc65bc7c815a5c9ee646c89a6770

But 2.4.6 and higher is the only thing we have to worry about in 2020.